pkgs/top-level/all-packages.nix

```diff
@@ -4951,7 +4951,7 @@ with pkgs;
 
   qosmic = libsForQt5.callPackage ../applications/graphics/qosmic { };
 
-  qownnotes = qt6Packages.callPackage ../applications/office/qownnotes {
+  qownnotes = callPackage ../by-name/qo/qownnotes/package.nix {
```

Comment:
AFAIK this should be removed when moving to by-name

Reply:
I thought those declarations should still be done there if needed (see https://github.com/NixOS/nixpkgs/tree/master/pkgs/by-name#changing-implicit-attribute-defaults). I saw many by-name packages in `pkgs/top-level/all-packages.nix`. 🤔